packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -16,6 +16,11 @@ export interface IDropdownProps extends ISelectableDroppableTextProps<HTMLDivEle
   placeHolder?: string;
 
   /**
+   * Custom width for dropdown. Otherwise the width of the input field is used.
+   */

```

Comment:
I believe 0 means default the width to the dropdown input box itself. Please update the documentation.